iothub/service/src/Common/Exceptions/DeviceMessageLockLostException.cs

```diff
@@ -7,10 +7,9 @@
 namespace Microsoft.Azure.Devices.Common.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when an attempt to communicate with a device fails
-    /// because the lock token was lost (if the connection is lost and regained for example).
-    /// This timeout has the same effect as if the message was abandoned.
+    /// This exception does not appear to be returned by the service for ServiceClient operations.
     /// </summary>
+    /// [Obsolete]
```

Comment:
Marking as obsolete as there seems to be no reason to get this exception in the ServiceClient.

Reply:
Why is the [obsolete] commented out like this?

Reply:
```suggestion
    [Obsolete("This exception does not appear to be returned by the service.")]
```

Reply:
This is what I meant to do but looks like I just did it in my head :D